src/test/java/com/faforever/client/game/PlayerCardTooltipControllerTest.java

```diff
@@ -66,8 +73,11 @@ public void testSetOther() {
         .defaultValues()
         .socialStatus(SocialStatus.OTHER);
     Player player = playerBuilder.get();
-    instance.setPlayer(player, 1000);
+    instance.setPlayer(player, 1000, Faction.NOMAD);
```

Comment:
Yea, not sure about this test.

Reply:
Well yes but let's not blame him I think it is ok. this whole faction number thing is fucked

Reply:
I'm just saying maybe don't make a test that asserts wrong behavior.

Reply:
I changed the enum name to RANDOM instead of NOMAD. It's not used anywhere else, so shouldn't be a problem.